src/pykeen/triples/triples_factory.py

```diff
@@ -916,11 +949,33 @@ def to_core_triples_factory(self) -> CoreTriplesFactory:
             metadata=self.metadata,
         )
 
-    def _get_binary_state(self):
-        return dict(
-            mapped_triples=self.mapped_triples,
+    def to_path_binary(self, path: Union[str, pathlib.Path, TextIO]) -> pathlib.Path:  # noqa: D102
+        path = super().to_path_binary(path=path)
+        # store entity/relation to ID
+        for name, data in dict(
             entity_to_id=self.entity_to_id,
             relation_to_id=self.relation_to_id,
+        ).items():
+            pd.DataFrame(data=data.items(), columns=["label", "id"],).sort_values(by="id").set_index("id").to_csv(
+                path.joinpath(f"{name}.tsv.gz"),
+                sep="\t",
+            )
+        return path
+
+    @classmethod
+    def _from_path_binary(cls, path: pathlib.Path) -> MutableMapping[str, Any]:
+        data = super()._from_path_binary(path)
+        # load entity/relation to ID
+        for name in ["entity_to_id", "relation_to_id"]:
```

Comment:
again might be good to make these names class variables since they're shared in the output function too